frontend/src/_components/SearchBox.jsx

```diff
@@ -53,7 +53,7 @@ export const SearchBox = forwardRef(
 
     return (
       <div className={`search-box-wrapper ${customClass}`}>
-        <div className="input-icon">
+        <div className="input-icon input-icon-padded">
```

Comment:
Isn't it useful to have it across the search boxes? It will be reusable.

Reply:
We can't implement it across all search boxes as it's used in multiple places and will require testing across all components using it.

Instead of this: 
can you please try doing something like this inside the targeted class in global-datasources.scss file?

Css:
.search-box-wrapper {
                .input-icon {
                    .form-control {
                        // add required css padding changes
                    }
                }
            }

Reply:
Agreed from the testing point of view. Will do the same and commit.